Tests/RosalindTests/__Snapshots__/RosalindAcceptanceTests/apple_ios_app.1.json

```diff
@@ -0,0 +1,59 @@
+{
+  "app" : {
+    "children" : [
+      {
+        "unknown" : {
```

Comment:
Should this be `directory` instead of `unknown`? I wonder if that would read better.

Reply:
Yeah! My plan was to add more types in follow-up PRs so I just did some basics here (i.e. it's either an app or unknown).

Reply:
I'd consider splitting `unknown` into `file` and `directory`. Anything should fall into these two categories and then we can have specialized types for others like `.app`.